libs/sdk-ui-tests-e2e/cypress/tools/export.ts

```diff
@@ -25,4 +32,60 @@ export class Export {
         // delete download folder before exporting
         this.deleteDownloadFolder(downloadsFolder);
     }
+
+    exportToCSV() {
+        cy.get(EXPORT_CSV_SELECTOR).click();
+        return this;
+    }
+
+    exportToXLSX() {
+        cy.get(EXPORT_XLSX_SELECTOR).click();
+        return this;
+    }
+
+    exportInsightOnWidgetByIndexToCSV = (index: number) => {
+        let widget = new Widget(index).waitChartLoaded();
+        new WidgetOptionsMenu(widget).open();
+        new ExportOption(".s-options-menu-bubble").getExportToCSVButtonElement();
+        messages.hasSuccessMessage("Export successful.");
+    };
+
+    exportInsightOnWidgetByIndexToXLSX = (index: number) => {
+        let widget = new Widget(index).waitChartLoaded();
+        new WidgetOptionsMenu(widget).open();
+        new ExportOption(".s-options-menu-bubble").getExportToXLSXButtonElement();
+        exportDialog.confirmExportXLSX();
+        messages.hasSuccessMessage("Export successful.");
+    };
+
+    expectExportedCSV(filename: string, contents: string) {
+        const downloadsFolder = Cypress.config("downloadsFolder");
+        cy.readFile(join(downloadsFolder, filename)).should("exist");
+        if (contents) {
+            cy.readFile(join(downloadsFolder, filename)).then(($el) => {
+                // remove " character on csv file then comparing the result
+                const updatedContents = $el.toString().replace(/"/g, "");
+                // eslint-disable-next-line jest/valid-expect
+                expect(updatedContents).to.contain(contents);
+            });
+        }
+        // delete download folder before exporting
+        this.deleteDownloadFolder(downloadsFolder);
+    }
+
+    expectExportedXLSX(filename: string, contents: string) {
```

Comment:
There is no difference between expectExportedCSV and expectExported XLSX, why do you separate to 2 method?

Reply:
XLSX đang fail trên SDK nên chưa check dc , function dc migrate từ AD vs KD qua

Reply:
In case migrate from AD/KD to UI SDK but if there is no difference, please custom/ remove it, don't use duplicate method on UI SDK (or even in AD/KD).